zio-http/src/main/scala/zhttp/http/Response.scala

```diff
@@ -1,16 +1,47 @@
 package zhttp.http
 
+import io.netty.handler.codec.http.HttpHeaderNames
 import zhttp.socket.{Socket, SocketApp, WebSocketFrame}
 import zio.Chunk
 
 import java.io.{PrintWriter, StringWriter}
+import java.time.Instant
 
 case class Response[-R, +E] private (
   status: Status,
   headers: List[Header],
   data: HttpData[R, E],
   private[zhttp] val attribute: HttpAttribute[R, E],
-)
+) { self =>
+
+  /**
+   * add cookies in response headers
+   */
+  def addCookie(cookie: Cookie): Response[R, E] =
```

Comment:
We could add cookies to attributes instead of headers. That way we will not have to serialize and create headers twice (one for `zhttp.http.Response` and one for `io.netty.codec.http.HttpResponse`.

Reply:
```suggestion
  def setCookie(cookie: Cookie): Response[R, E] =
```

This should overwrite the cookie everytime.

Reply:
- We can ignore the `Attribute` implementation for now.
- We will need a `getCookie` operator also for `Response` and `Request`.

Reply:
we can send multiple `set-cookie` headers

Reply:
It should be `getCookies` to get a list of cookies?

Reply:
Ok right let's not rename it to `setCookie`. Let's keep it as `addCookie`